learning_resources/etl/loaders.py

```diff
@@ -357,7 +363,10 @@ def load_program(program_data, blocklist, duplicates, *, config=ProgramLoaderCon
             },
         )
 
-    # if not created and not program.published:
+    if not created and not program.learning_resource.published:
+        search_index_helpers.deindex_program(program.learning_resource)
+    elif program.learning_resource.published:
+        search_index_helpers.upsert_program(program.learning_resource.id)
```

Comment:
Wondering if we should try using `pluggy` mentioned by @rhysyngsun in [this RFC](https://github.com/mitodl/hq/discussions/1638) to kick off indexing tasks?  Or maybe that requires more discussion and this is a viable fallback for now until that is finalized.

Reply:
I think we should try out pluggy but I don't want it to block this pr - maybe we can merge this for now and have a follow up pr that switches over to use pluggy?

Reply:
sounds good to me